cluster-autoscaler/config/const.go

```diff
@@ -32,4 +34,14 @@ const (
 	DefaultScaleDownUnreadyTimeKey = "scaledownunreadytime"
 	// DefaultMaxNodeProvisionTimeKey identifies MaxNodeProvisionTime autoscaling option
 	DefaultMaxNodeProvisionTimeKey = "maxnodeprovisiontime"
+	// DefaultIgnoreDaemonSetsUtilizationKey identifies IgnoreDaemonSetsUtilization autoscaling option
+	DefaultIgnoreDaemonSetsUtilizationKey = "ignoredaemonsetsutilization"
```

Comment:
This is only used in `aws` cloudprovider, shouldn't this const be moved there?

Reply:
+1

Reply:
@x13n as of now, yes but this is a generic option like [`DefaultScaleDownUnneededTimeKey`](https://github.com/search?q=repo%3Akubernetes%2Fautoscaler%20DefaultScaleDownUnneededTimeKey&type=code) which can be supported by multiple cloud providers in the future (not limited to AWS provider). 

I am fine with moving it to AWS provider since no other cloud provider is using it now but say azure provider wants to support this key tomorrow, they'd have to define their own constant with the same name (or maybe worse, a different name like `ignore-daemonset-utilization` thereby increasing the confusion for users) in their own cloud provider. 

So far it seems like we have kept such keys common across providers. If we want to keep them separate, I think that would be going against what we have been doing so far. It can be argued that every cloud provider should have freedom to define this key as they like but that means it opens up possibility for different cloudproviders naming this key differently. I am not sure if that's a good idea. 

Any feedback on this is welcome. If both of you think it's better to move this to AWS provider, I am happy to move the key to AWS provider.